src/Symfony/Bundle/FrameworkBundle/Command/ConfigDumpReferenceCommand.php

```diff
@@ -56,9 +59,8 @@ protected function configure(): void
   <info>php %command.full_name% framework</info>
   <info>php %command.full_name% FrameworkBundle</info>
 
-With the <info>--format</info> option specifies the format of the configuration,
-this is either <comment>yaml</comment> or <comment>xml</comment>.
-When the option is not provided, <comment>yaml</comment> is used.
+The <info>--format</info> option specifies the format of the configuration,
+this is either "{$helpFormats}".
```

Comment:
`either` is wrong here as you removed `or` after that in favor of a comma.

Reply:
Can you tell me what is the best here ?

(I'm not native speaker)